tests/namespace/test_plotting_namespace.py

```diff
@@ -0,0 +1,25 @@
+import importlib
+import pathlib
+
+import pytest
+
+from pyvista.core.errors import PyVistaDeprecationWarning
+
+namesapce_data = pathlib.Path(__file__).parent / 'namespace-plotting.txt'
```

Comment:
Just a style nitpick, but it might be better form to put this block of code in a function which gets called in `parametrize()` later. Unlikely that this test module will be imported, but still having this code in the global scope is a bit weird.

Reply:
I tend to let these sort of things slide in testing modules as no matter what, this code will have to run immediately on import or not. Maybe I wait until all PRs affecting this `namespace` testing directory are finished since this same logic repeats itself in three files now and make this reading/parameterizing one function in the `conftest`